tpchgen-arrow/Cargo.toml

```diff
@@ -9,9 +9,9 @@ readme = "README.md"
 license = "Apache-2.0"
 
 [dependencies]
-arrow = { version = "54.3.0", default-features = false, features = ["prettyprint"] }
+arrow = { version = "54.1.0", default-features = false, features = ["prettyprint"] }
 tpchgen = { path = "../tpchgen", version = "0.1.1" }
 
 [dev-dependencies]
-arrow-csv = "54.3.0"
-chrono = "0.4.39"
+arrow-csv = "54.1.0"
+chrono = "=0.4.39"
```

Comment:
Need the `=` to actually pin it otherwise its implicitly using a `^`

Reply:
I think it would be better if downstream crates / users pinned chrono (otherwise using this crate will force a particular version of chrono which doesn't seem quite right